### DISCUSSION TITLE
On mouse event, display a list.

### DISCUSSION THREAD

**Discussion by eunjin0212** [quasarframework/quasar#10909] [General - Components / Directives / etc] [answered]
Hello. I am doing web development through quasar framework. I want to create a mouse event. I want the following menu to be displayed when a mouse over event, not a mouse click event. I made this [codepen](https://codepen.io/jd-0001/pen/GRRVxLZ?editors=1010) as a reference, but it didn't work on my web. Help. 

What I want to know is 'how the mouseover event works' in vue3.0?

This is my Code

```html
<!-- html -->
<template>
  <q-btn
    label="MyBtnName"
    @mouseover="menuOver = true"
    @mouseleave="menuOver = false"
  >
    <q-menu v-model="menuOpen">
      <q-list>
        <q-item
          @mouseover="listOver = true"
          @mouseleave="listOver = false"
           >
          <q-item-section>
            {{ MyItems }}
          </q-item-section>
        </q-item>
      </q-list>
    </q-menu>
  </q-btn>
</template>
```

```javaScript
// script
import { ref } from 'vue';
import { debounce } from 'quasar';

export default {
  setup() {
    return {
      menuOpen: ref(false),
      menuOver: ref(false),
      listOver: ref(false),
    };
  },
  methods: {
    debounceFunc() {
      debounce(() => {
        this.checkMenu();
      }, 100);
    },
    checkMenu() {
      if (this.menuOver || this.listOver) {
        this.menuOpen = true;
      }
    },
  },
  watch: {
    mouseOver() {
      return this.debounceFunc();
    },
  },
};
```

**Comment by hawkeye64** [comment]
> but it didn't work on my web

What do you mean by this? The CodePen code should work with a bit of tweaking. Remember CodePen uses the UMD format of the Quasar library. Your project is likely using the ESM version of the library. Adjust accordingy.

**Reply by eunjin0212** [reply]
I made my code in the browser with the same configuration as the code pen. But `@mouseover` doesn't work.
That's my code. It's not exactly like a codepen, but it has a quasar cli installed. Excerpts from some of my code. 
The code pen is for reference only.

**Comment by pdanpdan** [comment]
I think you want mouseenter

**Reply by eunjin0212** [reply]
`@mouseenter` only triggers when the mouse enters the element on which it is set.
`@mouseover` triggers when the mouse enters the element or any of its children.
I want to `@mouseover` !!!! 😭

**Comment by pdanpdan** [comment, marked as answer]
You do it exactly the same, except in Vue3 you don't use `.native`
https://codepen.io/pdanpdan/pen/eYRqvRg?editors=1010
